tests/jballerina-unit-test/src/test/resources/test-src/reachability-analysis/reachability_analysis2.bal

```diff
@@ -0,0 +1,1034 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+function executeTestReachabilityWithIfContainingReturn() {
+    assertEqual(testReachabilityWithIfContainingReturn(), "Ballerina");
```

Comment:
If one of these fail the rest won't run, right? We used to write tests this way and switch to call individual functions using data providers (from the Java side) due to the same reason.

Reply:
Will update